include/fastdds/rtps/common/Locator.h

```diff
@@ -631,10 +651,20 @@ inline std::ostream& operator <<(
         std::ostream& output,
         const LocatorList_t& locList)
 {
-    for (auto it = locList.m_locators.begin(); it != locList.m_locators.end(); ++it)
+    output << "[";
+    if (locList.size() > 0)
     {
-        output << *it << ",";
+        output << *(locList.begin()) ;
+        for (auto it = locList.m_locators.begin()+1; it != locList.m_locators.end(); ++it)
```

Comment:
Consider changing this loop into a ranged for.

Reply:
As it starts in the second element, there is not a simple way to change this into a ranged for.